feature-group-definitions/calc-constants.yml

```diff
@@ -0,0 +1,9 @@
+name: calc() constants
```

Comment:
@ddbeck can you review this name and description before I change them all? I found it hard to write the description in an active voice, since a constant is pretty darn passive, it is used but does not have behavior. How do you think we should phrase that?

Reply:
Yeah, this is a place where the passive voice is genuinely OK.

> The `e`, `pi`, `infinity`, and `NaN` constants are accepted by CSS `calc()` expressions.

By comparison, the active version puts the wrong information first:

> `calc()` CSS function expressions accept the constants `e`, `pi`, `infinity`, and `NaN`.

Functions are easier to write actively, but also work passively:

> The `min()`, `max()`, and `clamp()` functions compare CSS values.
> `sin()`, `cos()`, … `atan2()` are trigonometry functions for CSS values.